src/shapes/text.class.js

```diff
@@ -1512,6 +1512,15 @@
       this.callSuper('render', ctx);
     },
 
+    /**
+     *
+     * @param {string} value
+     * @returns {string[]} array of graphemes
+     */
+    graphemeSplit: function (value) {
+      return fabric.util.string.graphemeSplit(value);
+    },
```

Comment:
this feels extra, but i have zero issues with it.
Why not swapping fabric.util.string.graphemeSplit directly?

Reply:
if this is meant to ease overrides, please let's add a comment here to suggest that to developers

Reply:
> i think i because we want to know immediately which is the large word of all text before we take decisions on intermediate long words

Absolutely!

Reply:
[f6864cb](https://github.com/fabricjs/fabric.js/pull/7806/commits/f6864cba191c8aea1629ab4dc86c9377465e8ffa)